### PR TITLE
Fix version inconsistencies in mathml/*

### PR DESCRIPTION
**Pull request by vinyldarkscratch**
Sub-PR of #3368 for the fixes of the files themselves.

### CODE REVIEW

File: mathml/elements/mphantom.json [hunk 1]
```diff
@@ -36,7 +36,7 @@
               "version_added": "6"
             },
             "safari_ios": {
-              "version_added": false
+              "version_added": "6"
```
Comment (Elchi3) [inline mathml/elements/mphantom.json]: I think this should remain `false`, and the subfeature should be `false`, too.
https://developer.mozilla.org/en-US/docs/Web/MathML/Element/mphantom#Browser_compatibility

File: mathml/elements/mrow.json [hunk 2]
```diff
@@ -36,7 +36,7 @@
               "version_added": "6"
             },
             "safari_ios": {
-              "version_added": false
+              "version_added": "6"
```
Comment (Elchi3) [inline mathml/elements/mrow.json]: Same here, should remain `false` and sub features should be `false`, too.
https://developer.mozilla.org/en-US/docs/Web/MathML/Element/mrow#Browser_compatibility

File: mathml/elements/mroot.json [hunk 3]
```diff
@@ -36,7 +36,7 @@
               "version_added": "6"
             },
             "safari_ios": {
-              "version_added": false
+              "version_added": "6"
```
[the remark this answers is not in the document]
Comment (vinyldarkscratch) [reply mathml/elements/mroot.json]: I think that after a while, I started changing the parent on accident, haha -- fixed!

**Comment by vinyldarkscratch** [comment]
Fixed all the changes as requested.  :wink:

**Pull request merged**